src/python/WMQuality/Emulators/DBSClient/DBSReader.py

```diff
@@ -9,19 +9,19 @@
 
 class _MockDBSApi():
     """Mock dbs api"""
-    def __init__(self, args):
```

Comment:
args are dict type below line 68, but DbsApi first argument is url str. which gives the error. 
Also,  DbsApi init signature is (url="", proxy=None, key=None, cert=None, verifypeer=True, debug=0, ca_info=None, userAgent="") so I just to want to make compatible.